governance.md

```diff
@@ -0,0 +1,57 @@
+
+The BDFL can appoint his/her successor, but it is expected that the Steering Council would be consulted on this decision. If the BDFL is unable to appoint a successor, the Steering Council will make a suggestion or suggestions to the main NumFOCUS Board. While the Steering Council and main NumFOCUS Board will work together closely in selecting a BDFL, the main NumFOCUS Board will make the final decision.
+
+### Steering Council
+
+The Project will have a Steering Council that is elected to further the overall mission of the The Project. The initial Steering Council was created by the BDFL, however, future membership rules will determined by the Steering Council itself. The overall role of the Steering Council is to ensure, through working with the BDFL and taking input from the community, the long-term well-being of the project, both technically, editorially, and as a community.
+
+The Steering Council and its Members play a special role in certain situations. In particular, the Council may:
+
+- Make decisions about the overall scope, vision, and direction of The Project.
+- Make decisions about strategic collaborations with other organizations or individuals.
+- Make decisions about new starting a new journal (or incorporating an existing journal) under the Open Journals organization.
+- Make decisions about the Services that are run by The Project and manage those Services for the benefit of the Project and Community.
+- Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+
+#### Steering Council membership
+
+To be written by the Council.
+
+#### Subcommittees
+
+Question: Does each journal need its own subcommittee or is this effectively the editorial board of that journal?
+
+#### NumFOCUS Subcommittee
+
+The Council will maintain one narrowly focused subcommittee to manage its interactions with NumFOCUS.
+
+- The NumFOCUS Subcommittee is comprised of 5 persons who manage project funding that comes through NumFOCUS. It is expected that these funds will be spent in a manner that is consistent with the non-profit mission of NumFOCUS and the direction of the Project as determined by the full Council.
+- This Subcommittee shall NOT make decisions about the direction, scope, or technical direction of the Project.
+- This Subcommittee will have 5 members, 4 of whom will be current Steering Council members and 1 of whom will be external to the Steering Council.
```

Comment:
So, this is a separate subcommittee from the Steering Council? And it primarily does financial oversight and planning? I'm not quite sure why it's set up this way, rather than a responsibility of the Steering Council or as a subcommittee of the Steering Council, or a Treasurer. Is it that this committee will do the day-to-day work, and you want to separate out that responsibility? Totally understand there could be a reason here, just thinking this could make things more difficult to have operationally aligned.

Reply:
Also, rather than 'NumFOCUS Subcommittee' a more generalizable name would be good for governance names, like 'Fiscal Subcommittee' or 'Operations Subcommittee'

Reply:
> Totally understand there could be a reason here, just thinking this could make things more difficult to have operationally aligned.

There actually isn't. This is language I lifted directly from the Jupyter governance documents.

Perhaps the fiscal subcommittee/NumFOCUS subcommittee should simply be a subset of the main steering council?

Reply:
I think having it be a subset of the main steering council makes the most sense. I suggested a line about fiscal oversight in the steering committee role, so it could be included there. Then the Steering Council could decide if they want a subcommittee or a Treasurer to have this role.